prov/verbs/src/ep_rdm/verbs_rdm_cm.c

```diff
@@ -615,21 +617,76 @@ ssize_t fi_ibv_rdm_conn_cleanup(struct fi_ibv_rdm_conn *conn)
 	return ret;
 }
 
+static int fi_ibv_rdm_poll_cq(struct fi_ibv_rdm_ep *ep)
```

Comment:
Seems like this function almost duplicates `fi_ibv_rdm_tagged_poll_send`. Is it possible to merge them (or refactor existing one) adding `util_buf_release(fi_ibv_rdm_request_pool, request);` instead of `assert(0)` in `wc_error` section to avoid code duplication?